### DISCUSSION TITLE
More payment method

### DISCUSSION THREAD

**Discussion by bruno5011** [gameserverapp/Platform#128] [Suggestions]
Add more payment way on community site, currently we only have paypal.

**Comment by mvaessen** [comment]
Looking forward to more specific suggestions, namely parties that you like to see integrated.

**Reply by henrik123098** [reply]
We get Alot of paysafecard support request on our community.

**Comment by mvaessen** [comment]
Here are a couple options I consider implementing. I will only consider implementing payment providers, because the alternative is having to integrate each payment option individually. Any further suggestions are welcome (include docs with your suggestion).

- https://stripe.com/en-nl/pricing#pricing-details
- https://www.mollie.com/en/pricing

**Reply by henrik123098** [reply]
the mollie one seems nice its has soo many options

**Reply by DemonSlayer111** [reply]
Stripe support would be fantastic. I've had very bad experiences using PayPal so any alternative is a plus for me.

**Reply by Leonrrcs** [reply]
Booth would be awsome ^-^

**Reply by Ian-Highlander** [reply]
Another "vote" for Stripe here if you could ever make that happen please that would be amazing. Paypal are awful and causing all sorts of issues for us.

**Reply by mvaessen** [reply]
Stripe support was added today. Check out Discord for more information!
